docs/_content/standaard/verzoeken/index.md

```diff
@@ -0,0 +1,46 @@
+# Inleiding
+Een verzoek is een aanvraag of opdracht aan de gemeente voor de levering van een product of dienst. Verzoeken vormen dus de schakel tussen de klant, producten en diensten van de gemeente zoals in de PDC staan, en het zaakgericht werken voor de afhandeling. Dit biedt een aantal voordelen, onder andere op het gebied van herkenbaarheid voor de klant, die niet meer wordt geconfronteerd met voor hem onherkenbare zaaktypen. Voor gemeenten ontstaat bovendien meer flexibiliteit: een aanvraag van een klant leidt niet langer vanzelfsprekend tot een zaak, en één of meerdere verzoeken kunnen worden gerelateerd aan één of meerdere zaken.
```

Comment:
> onder andere op het gebied van herkenbaarheid voor de klant, die niet meer wordt geconfronteerd met voor hem onherkenbare zaaktypen.
Ik zou dit weg laten, ook zonder verzoeken moeten we een klant namelijk niet lastigvallen met onherkenbare zaaktypen.

Reply:
Eens. Ik zou er dan van willen maken "die niet geconfronteerd wordt met concepten uit de interne bedrijfsvoering van de gemeente"